### DISCUSSION TITLE
How to have the most recent mermaid version?

### DISCUSSION THREAD

**Discussion by sanjidnet** [cotes2020/jekyll-theme-chirpy#1205] [Q&A] [answered]
I am try to render a new Mermaid visualization, `mindmap`
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/4423129/db7dcea7-04ef-4507-84ab-dfbcd543e8bf)

but is is not working. 
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/4423129/59083c67-c77b-4b92-93e7-2b909d670275)

Could it be because Jekyll / Github / Github Pages is not updated to incorporate the most recent Mermaid version?
Is there any way to validate that, i.e. find out which version of mermaid is used to render the visualization?
Anybody know where I should be raising this?

**Comment by NichtsHsu** [comment, marked as answer]
See #994

**Reply by sanjidnet** [reply]
@NichtsHsu thanks! I should've checked there first. 
@cotes2020 mindmaps are really cool though. And mermaid has been adding more visualizations since upgrade to 10.x.x. Do you plan to keep mermaid at 9.3? 
Thanks for the template by the way. 
It's been my website for a couple of years now.

**Comment by cotes2020** [comment]
@sanjidnet Just wait for the next release of Chirpy.
